### PR TITLE
Improve MAVEN build Performance

### PR DESCRIPTION
**Pull request by ChenZhangg**
[Parallel builds in Maven 3](https://cwiki.apache.org/confluence/display/MAVEN/Parallel+builds+in+Maven+3) Maven 3.x has the capability to perform parallel builds.

[Apache Maven Dependency Plugin](https://maven.apache.org/plugins/maven-dependency-plugin/index.html) can be used to find unused dependencies. And I found following list. Maybe we can remove them.
findsecbugs-root-pom
findsecbugs-test-util
{groupId='org.hamcrest', artifactId='hamcrest-core', version='2.1', scope='test'}
findsecbugs-samples-deps
findsecbugs-samples-kotlin
{groupId='org.jetbrains.kotlin', artifactId='kotlin-stdlib-jdk8', version='1.2.51', scope='compile'}
findsecbugs-samples-java
findsecbugs-samples-jsp
findsecbugs-plugin
{groupId='com.h3xstream.findsecbugs:findsecbugs-samples-kotlin', artifactId='test-jar', version='1.12.0-SNAPSHOT', scope='test'}
{groupId='com.h3xstream.findsecbugs:findsecbugs-samples-java', artifactId='test-jar', version='1.12.0-SNAPSHOT', scope='test'}
{groupId='com.h3xstream.findsecbugs:findsecbugs-samples-jsp', artifactId='test-jar', version='1.12.0-SNAPSHOT', scope='test'}
{groupId='ch.qos.logback', artifactId='logback-classic', version='1.2.3', scope='test'}
{groupId='commons-io', artifactId='commons-io', version='2.6', scope='test'}
{groupId='com.h3xstream.findsecbugs', artifactId='findsecbugs-samples-deps', version='1.12.0-SNAPSHOT', scope='provided'}

According to [Maven parallel test](https://www.baeldung.com/maven-junit-parallel-tests), we can run tests in parallel.

=====================
If there are any inappropriate modifications in this PR, please give me a reply and I will change them.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1,374 +1,375 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```
Comment (h3xstream) [inline pom.xml:1]: The complete reformatting makes it impossible to review.

**Review by h3xstream — CHANGES_REQUESTED** [review]
This looks like an auto-generated PR. It has to much noise in the `pom.xml` to do a review.

**Comment by ChenZhangg** [comment]
I will reformat it

**Comment by h3xstream** [comment]
⚠️ 
This PR seems very low quality as no test were made on the contributor part.
It is only a blind automate change and it assume that others will take time to make it works.

**Comment by h3xstream** [comment]
The change was not test. The build is failling because of an obvious typo in the `pom.xml`.
It is pretty clear that the author blindly made the change without any interest to validate its change.

**Pull request closed without merging**